includes/general.php

```diff
@@ -1052,7 +1052,16 @@ function pods_shortcode_run( $tags, $content = null ) {
 
 		return $pod->form( $tags['fields'], $tags['label'], $tags['thank_you'] );
 	} elseif ( ! empty( $tags['field'] ) ) {
-		if ( empty( $tags['helper'] ) ) {
+		$template = ! empty( $tags['template'] ) ? $tags['template'] : $content;
+
+		if ( $template ) {
+			$return  = '';
+			$related = $pod->field( $tags['field'], array( 'output' => 'find' ) );
+
+			if ( $related instanceof Pods && $related->valid() ) {
+				$return .= $related->template( $tags['template'], $template );
```

Comment:
@JoryHogeveen Can you describe the usage here, if `$tags['template']` is set then it sets `$template` too.

Reply:
Good one, that isn't correct.. Weird that it worked locally though..